src/locales/messages.pot

```diff
@@ -3437,6 +3446,9 @@ msgstr ""
 msgid "When enabled, the project will hold the fee amount in ETH instead of the fees being processed automatically. <0>Learn more.</0>"
 msgstr ""
 
+msgid "When enabled, transfers will be paused."
```

Comment:
...project token transfers will be paused. This does not apply to ERC-20 tokens if issued.